projects/ddata-ui-input/src/lib/components/date/date-input.component.html

```diff
@@ -13,7 +13,6 @@
     <ng-container *ngIf="!isViewOnly">
       <input
         ngxDaterangepickerMd
-        type="text"
```

Comment:
If you remove the `type="text"` paramter and not define other one instead of this, it means fallback to default which is `type="text"`. So please, don't remove this.